test/hotspot/jtreg/runtime/appcds/MoveJDKTest.java

```diff
@@ -42,19 +40,21 @@
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.nio.file.StandardCopyOption;
+import jdk.test.lib.cds.CDSTestUtils;
 import jdk.test.lib.process.OutputAnalyzer;
```

Comment:
Includes the changes from [JDK-8251213](https://bugs.openjdk.org/browse/JDK-8251213) ( commit [c1093dc](https://github.com/openjdk/jdk/commit/c1093dc2447c39c0bd1e1db3627dd353662a5040) ) on this file